mm2src/mm2_bin_lib/build.rs

```diff
@@ -51,31 +51,27 @@ fn mm_version() -> String {
     // and a release tag when building from some kind of a stable branch,
     // though we should keep the ability for the tooling to provide the “MM_VERSION”
     // externally, because moving the entire ".git" around is not always practical.
-    let mut version = "UNKNOWN".to_string();
-    let mut command = Command::new("git");
-    command.arg("log").arg("--pretty=format:%h").arg("-n1");
-    if let Ok(go) = command.output() {
-        if go.status.success() {
-            version = from_utf8(&go.stdout).unwrap().trim().to_string();
-            if !Regex::new(r"^\w+$").unwrap().is_match(&version) {
-                panic!("{}", version)
-            }
-        }
-    }
+    let mut version = env!("CARGO_PKG_VERSION").to_owned();
 
     let mm_version_p = root().join("../../MM_VERSION");
     let v_file = String::from_utf8(slurp(&mm_version_p)).unwrap();
-    let v_file = v_file.trim().to_string();
-    // if there is no MM_VERSION file there is no need to create it
+    let tag_from_ci = v_file.trim().to_string();
+
+    // if there is MM_VERSION file, that means CI wants to put a tag to version
     if !v_file.is_empty() {
-        if !v_file.contains(&version) {
-            // If the file doesn't contain the latest commit hash then this is a local build
-            // and the env version variable should be "v_file"_"version" as "v_file" is the version from the local file
-            // and "version" is the latest commit hash
-            version = format!("{}_{}", v_file, version);
-        } else {
-            // If the file contains the latest commit hash then this is a CI build and the version generated by CI should be written to env
-            version = v_file;
+        version = format!("{}_{}", version, tag_from_ci);
```

Comment:
I think it is cheaper to execute the `.trim().to_string()` op directly in this line as opposed to returning/assigning it to a variable in L58 since `tag_from_ci` is only refd one time.